docs/pages/examples.md

```diff
@@ -0,0 +1,131 @@
+
+ - **Code**:
+    ```rust
+    use core::felt252;
+
+    fn main() -> Array<felt252> {
+        let n = 10;
+        let mut result = array![];
+        result.append(fib(1, 1, n));
+        result
+    }
+
+    fn fib(a: felt252, b: felt252, n: felt252) -> felt252 {
+        match n {
+            0 => a,
+            _ => fib(b, a + b, n - 1),
+        }
+    }
+    ```
+  - **Explanation**:
+    - `felt252`: A type in Cairo that represents field elements, where values are limited by a modulus of 2^252.
+    - The `main` function initializes `n` (the number of Fibonacci terms) and starts the Fibonacci sequence with the initial values `a = 1` and `b = 1`.
+    - The recursive `fib` function computes Fibonacci numbers. The base case returns `a` when `n` is 0, while the recursive case reduces `n` by 1 and swaps the values of `a` and `b` until the result is computed.
+
+
+### 2. Execution Artifacts
+Execution artifacts include the trace and memory files generated when running the Cairo programs.
+
+- **Trace**: [`fibonacci_trace.json`](https://github.com/dipdup-io/stone-packaging/tree/master/test_files/fibonacci/fibonacci_trace.json)  
+  - **Description**: A detailed step-by-step execution log of the Fibonacci program. This file provides insight into the program’s execution, making it a key tool for debugging and verification.
+  
+- **Memory**: [`fibonacci_memory.json`](https://github.com/dipdup-io/stone-packaging/tree/master/test_files/fibonacci/fibonacci_memory.json)  
+  - **Description**: Shows the memory state during execution, including how `felt252` values are stored and manipulated.It records how variables are stored and changed throughout the run, helping developers ensure proper memory handling.
+
+- **Input Files**:
+  - **Private Input**: [`fibonacci_private_input.json`](https://github.com/dipdup-io/stone-packaging/tree/master/test_files/fibonacci/fibonacci_private_input.json)  
+    - **Description**: Contains cryptographic inputs like Pedersen hashes and range checks and other sensitive data used during the proof generation process.  
+    - **Content**:
+      ```json
+      {
+          "trace_path": "./fibonacci_trace.json",
+          "memory_path": "./fibonacci_memory.json",
+          "pedersen": [],
+          "range_check": [],
+          "ecdsa": []
+      }
+      ```
+
+  - **Public Input**: [`fibonacci_public_input.json`](https://github.com/dipdup-io/stone-packaging/tree/master/test_files/fibonacci/fibonacci_public_input.json)  
+    - **Description**: Lays out the public input values required for running the Fibonacci program. This file ensures the correct parameters, such as the number of steps and memory segments, are applied.
```

Comment:
Сan you please provide a link to the website where you get this information from?

Reply:
> Сan you please provide a link to the website where you get this information from?
@dmirgaleev 
i used the referenced files provided in the task description.
Please can you give me a detailed overview of what you want to see?

Reply:
I'm just trying to understand if the docs you provided are correct. For instance, where can I find, that `fibonacci_trace.json`  contains "A detailed step-by-step execution log of the Fibonacci program. This file provides insight into the program’s execution, making it a key tool for debugging and verification."?

Reply:
> I'm just trying to understand if the docs you provided are correct. For instance, where can I find, that `fibonacci_trace.json` contains "A detailed step-by-step execution log of the Fibonacci program. This file provides insight into the program’s execution, making it a key tool for debugging and verification."?

Yea. i am gonna correct that. Maybe i will provided technical details of what the files does instead of making the an abstract explanation.